docs/data-management/qaqc-criteria.md

```diff
@@ -0,0 +1,196 @@
+# QA/QC criteria
+
+The following lists the pre-defined exclusion criteria for analyses of whole-brain structural and functional connectomes.
+
+## Anatomical MRI
+
+Given our planned analysis, the T1w image will be used for the spatial alignment with the standard MNI152NLin2009cAsym template.
+In addition, surface reconstructions from the T1w image will guide the co-registration of structural and functional (BOLD) images in fMRIPrep.
+Since the latter preprocessing steps are relatively robust to structural images with mild artifacts, the exclusion criteria for unprocessed T1w images are lenient.
+However, individual T1w images may be excluded without such a decision lead to exclusion of the whole session in which it belongs.
+We annotate subjects with visible artifacts in the T1w images in order to ensure rigorous scrutinizing of spatial normalization and surface reconstruction outputs from fMRIPrep (if both modalities passed the first QC checkpoint with MRIQC).
```

Comment:
This is good -- I think this part is important to lay out how images will be assessed.

One important thing to bear in mind: we need to be clear on the "exclusion level". Individual T1w or T2w images can be excluded if they are really bad, but that does not lead to excluding the full session. That is in contrast to fMRI tasks and rest, as well as DWI, where an instance of low quality that is excluded means the full session is excluded.